runelite-client/src/main/java/net/runelite/client/callback/Hooks.java

```diff
@@ -103,6 +106,8 @@ public static void clientMainLoop(Client client, boolean arg1)
 		{
 			shouldProcessGameTick = false;
 			eventBus.post(tick);
+
+			_deferredEventBus.post();
```

Comment:
I'd want the GameTick event to run after the events in the deferred eventbus. Otherwise if a plugin caches certain NPCs/Players/Items etc. after receiving them from the respective event, they would not appear to be there in the GameTick event until 1 tick later.